zipkin-server/README.md

```diff
@@ -268,6 +268,16 @@ The following apply when `STORAGE_TYPE` is set to `elasticsearch`:
                            to 0 as it would mean a machine failure results in data loss.
     * `ES_USERNAME` and `ES_PASSWORD`: Elasticsearch basic authentication, which defaults to empty string.
                                        Use when X-Pack security (formerly Shield) is in place.
+    * `ES_CREDENTIALS_FILE_PATH`: Elasticsearch credentails properties file path, which defaults to empty
+                               string. The suffix of file should be `.properties`. The keys of
+                               `username` and `password` are `zipkin.storage.elasticsearch.username`
+                               and `zipkin.storage.elasticsearch.password` respectively.
+                               Before the credentials are loaded from the file, `ES_USERNAME` and
+                               `ES_PASSWORD` is in place. If they are empty, basic authentication
+                               is disable temporarily. This file is reloaded periodically, which uses
+                               `ES_CREDENTIALS_FILE_REFRESH_INTERVAL_IN_SECOND` as the interval.
+    * `ES_CREDENTIALS_FILE_REFRESH_INTERVAL_IN_SECOND`: Credentials file refresh interval in seconds
```

Comment:
Looking at other time related configuration options `SELF_TRACING_FLUSH_INTERVAL`, `ES_TIMEOUT` , `QUERY_LOOKBACK` etc we don't specify the time unit it seems. Would it be ok for you to leave it out here as well, so `ES_CREDENTIALS_FILE_REFRESH_INTERVAL` ?

Reply:
this is also chatty ES_CREDENTIALS_FILE_INTERVAL we don't add units anywhere

Reply:
change my mind. I like @jorgheymans as it matches conventionally `SELF_TRACING_FLUSH_INTERVAL`

Reply:
actually `ES_CREDENTIALS_REFRESH_INTERVAL` is fine as there's no ambiguity about the target of the refresh

Reply:
done